.github/workflows/update-translations.yml

```diff
@@ -1,10 +1,6 @@
 name: Update texts for translators
 
 on:
-  push:
-    branches:
-      - 'release-**'
```

Comment:
Surely this meant that (non-scheduled runs) were only being run for pushes to branches that were called `release-` something - and those do not crop up except when a new version is pending... 😕

Reply:
That's right. But as I've been cherrypicking into the release branch often, it was causing a lot of runs.

Reply:
So it would not be good to make a release on a Thursday as the files could be really out of date? 😉 

🤔 I have been using this snippet:
```diff
 jobs:
   compile-mudlet:
     name: ${{matrix.buildname}}
+    # From: https://stackoverflow.com/a/59775665/4805858
+    if: "!contains(github.event.commits[0].message, '[skip gha]')"
     runs-on: ${{matrix.os}}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
```
to try and skip running the GitHub actions (and Travis) whilst I am hacking on the Windows CI - could this approach be used to run the text update on demand as well as on a schedule - say with an entry `[update translations]` magic string in the (top line of the) commit message?

Reply:
I think just removing the build from automatically running on new commits is fine. We can always run it manually to make sure it's up to date: https://wiki.mudlet.org/w/Manual:Github_Actions